cmake/mysql_version.cmake

```diff
@@ -18,7 +18,7 @@
 #
 
 SET(SHARED_LIB_MAJOR_VERSION "18")
-SET(SHARED_LIB_MINOR_VERSION "1")
+SET(SHARED_LIB_MINOR_VERSION "2")
```

Comment:
I'm not sure about 18.2, usually version bump depends on ABI incompatibility. @laurynas-biveinis it's OK if it's compatible with previous version except new option, could you please confirm?

Reply:
NET is not a part of public API unless I'm missing something. We might be OK without a version bump, will figure out in due course